rosidl_generator_cpp/resource/msg__struct.hpp.template

```diff
@@ -133,26 +133,19 @@ for field in spec.fields:
 @[end for]@
 
   // pointer types
-  typedef @(cpp_full_name)<ContainerAllocator> *
-    RawPtr;
-  typedef const @(cpp_full_name)<ContainerAllocator> *
-    ConstRawPtr;
-  typedef std::shared_ptr<@(cpp_full_name)<ContainerAllocator>>
-    SharedPtr;
-  typedef std::shared_ptr<@(cpp_full_name)<ContainerAllocator> const>
-    ConstSharedPtr;
-
-  typedef std::unique_ptr<@(cpp_full_name)<ContainerAllocator>>
-    UniquePtr;
-  typedef std::unique_ptr<@(cpp_full_name)<ContainerAllocator> const>
-    ConstUniquePtr;
-
-  typedef std::weak_ptr<@(cpp_full_name)<ContainerAllocator>>
-    WeakPtr;
-  typedef std::weak_ptr<@(cpp_full_name)<ContainerAllocator> const>
-    ConstWeakPtr;
+  using RawPtr = @(cpp_full_name)<ContainerAllocator> *;
+  using ConstRawPtr = const @(cpp_full_name)<ContainerAllocator> *;
+  using SharedPtr = std::shared_ptr<@(cpp_full_name)<ContainerAllocator>>;
+  using ConstSharedPtr = std::shared_ptr<@(cpp_full_name)<ContainerAllocator> const>;
```

Comment:
Since `@(cpp_full_name)` will tend to be long, I'd agree wrapping is not a bad idea. But just for reference, our line length limit is 100 characters.